react-context/my-app/src/components/useCart.ts

```diff
@@ -0,0 +1,9 @@
+import { useContext } from 'react';
+import { CartContext } from './CartContext';
+
+export function useCart() {
```

Comment:
You should provide an explicit return type here. That will allow you to tell TypeScript that this function will _not_ return `undefined`.